src/main/java/uk/gov/ons/ssdc/supporttool/model/entity/UserGroupAuthorisedActivityType.java

```diff
@@ -23,4 +23,5 @@
   CREATE_CASE_REFUSAL,
   CREATE_CASE_INVALID_ADDRESS,
   CREATE_CASE_FULFILMENT,
+  UPDATE_SAMPLE_SENSITIVE
```

Comment:
This will need to be added to Case Processor, Case API and ROps too.

(Meant to put this comment here, not the other place)

Reply:
This hasn't been done

Reply:
@NickGrantONS I've opened:
https://github.com/ONSdigital/ssdc-rm-caseprocessor/pull/41
https://github.com/ONSdigital/ssdc-rm-response-operations/pull/4
https://github.com/ONSdigital/ssdc-rm-case-api/pull/27